plugins/CoreUpdater/CoreUpdater.php

```diff
@@ -52,6 +53,10 @@ public static function getComponentUpdates(PiwikCoreUpdater $updater)
 
     public function dispatch()
     {
+        if (!SettingsPiwik::isAutoUpdateEnabled()) {
```

Comment:
About this one I'm not 100% sure. I think it is excepted to not show the "Please update screen" when auto update is disabled since a user would perform the update through the console?

The problem is that when we update code eg on Cloud, we always have to add a `return;` statement here hard coded during deployment because otherwise for short time the update screen would be visible. We don't want this screen to be visible though since we're performing the DB update through the console...

Reply:
Makes sense to me.